### ISSUE TITLE
Ctrl+click to follow link

### ISSUE THREAD

**Issue by borekb** [careteditor/caret#709]
Ctrl+click on links was a nice feature in v3 which seems to be missing from v4. Any plans to bring it back?

**Comment by astoilkov** [comment]
Hi Borek,

We actually changed the combination to `Alt+click`. This decision was made so we are more familiar to new users because most apps use `Ctrl+click` for adding a cursor. Also, apps like VS Code use `Alt+click` for opening links.

Hope you understand our decision.

**Comment by borekb** [comment]
I didn't know other editors use Ctrl+click for multi-cursor: VSCode uses Alt by default and when I press Ctrl over a link in a Markdown document, I see this:

![image](https://user-images.githubusercontent.com/101152/38077892-dfb298be-333a-11e8-9b5e-2a563bd48126.png)

But I learned that it's supposedly a feature in Atom and Sublime Text. On the other hand, Ctrl+click is a common convention in MS Office, Caret v3 :) and elsewhere.

By the way, do you think, or have evidence, that people actively use multiple cursors in Caret? I was quite surprised to find this advanced features there – pleasantly surprised I have to say because it shows that Caret is not "yet another MD editor" but at the same time, I never used it once in practice. My gut feeling would be to not prioritize this over following links.

**Comment by astoilkov** [comment]
Hmm. That's strange. At our computers (Windows and Mac) VS Code uses `Alt+click` to follow links. Are you sure you haven't installed a keybinding or an extension that alters that?

**Comment by borekb** [comment]
Yes, I'm pretty sure. It's controlled by `editor.multiCursorModifier` option which is `alt` by default (so by default, following links is Ctrl+click). Just tried on an extension-less VSCode installation.

**Comment by astoilkov** [comment]
You are totally right. Thanks a lot for the feedback. We will figure out what we are going to do and write back here.

**Comment by borekb** [comment]
I'm not sure if I remember correctly but wasn't Ctrl+click working in 4.0.0-rc5? I'm now in rc8 and Ctrl+click (or Alt+click) doesn't do anything so I'm not sure if this is a regression in rc8 or it has always been like that in the betas.
